app/assets/stylesheets/rooms.scss

```diff
@@ -1,20 +1,40 @@
 #room-card {
   border: 2px solid transparent !important;
-  width: 311px;
+  width: 305px;
+
+  @include media-breakpoint-down(xxl) {
+    width: 261px;
+  }
+
+  @include media-breakpoint-down(xl) {
+    width: 216px;
+  }
+
+  @include media-breakpoint-down(md) {
+    width: 245px;
+  }
+
+  @include media-breakpoint-down(sm) {
+    min-width: 245px;
+    width: 100%;
```

Comment:
Why not width 100% them and then set the width using the row and column bootstrap system?

Reply:
It feels better with a fixed width.

Having them at 100% will stretch out the card, is unpredictable.

The card should be more of a square than a rectangle.